localizer/src/detection.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import cv2
+
+# Model
+try:
+    model = torch.hub.load('ultralytics/yolov5', 'yolov5s', pretrained=True).eval()
+except Exception as e:
+    print(e)
+    print('offline mode')
+    model = torch.hub.load('/Users/john/.cache/torch/hub/ultralytics_yolov5_master', 'yolov5s', source='local', pretrained=True).eval()
```

Comment:
TODO: remove this hard-coding

When connected to the Tello WiFi, you're offline, so model-loading needs to use cache.